core/src/main/java/com/googlecode/psiprobe/beans/ContainerWrapperBean.java

```diff
@@ -99,9 +99,8 @@ public void setWrapper(Wrapper wrapper) {
                   tomcatContainer = (TomcatContainer) obj;
                   tomcatContainer.setWrapper(wrapper);
                   break;
-                } else {
-                  logger.debug("Cannot bind " + className + " to " + serverInfo);
                 }
+                logger.debug("Cannot bind " + className + " to " + serverInfo);
```

Comment:
I believe there is a little more to it.  Basically, the conditional as a whole is not completely intact because it breaks out of the conditional.  So one Eclipse check and I think others state this is bad.  Of course those checkers go on further to state that we shouldn't have multiple exit points (ie break, continue, return, etc).  One step at a time.  I don't think Eclipse goes far enough though as it only mindlessly suggests these fixes.  I know Intellij sort of layers the changes to the point the developer actually learns something in the process.  At the moment, I'm just taking the mindless approach :)  Later, deep diving using Intellij will become necessary.